engine/src/flutter/impeller/renderer/backend/gles/texture_gles.cc

```diff
@@ -45,15 +45,18 @@ static bool IsDepthStencilFormat(PixelFormat format) {
 }
 
 static TextureGLES::Type GetTextureTypeFromDescriptor(
-    const TextureDescriptor& desc) {
+    const TextureDescriptor& desc,
+    bool supports_implict_msaa) {
```

Comment:
nit: I think sending in the Capabilities would be a better interface.  It's more clear where people are getting this for and what the purpose is.

Reply:
Done